src/SmartComponents/Advisor/Advisor.js

```diff
@@ -152,12 +156,20 @@ const Advisor = () => {
                             <FlexItem>
                                 <Flex
                                     alignItems={{ default: 'alignItemsFlexCenter' }}
-                                    justifyContent={{ default: 'justifyContentCenter' }}>
-                                    {advisorIncidents?.meta?.count > 0 &&
-                                        <ExclamationCircleIcon className='pf-v5-u-font-size-xl pf-v5-u-danger-color-100' />}
-                                    <span className='pf-v5-u-font-size-2xl pf-v5-u-text-align-center pf-v5-u-font-weight-normal'>
-                                        {intl.formatMessage(messages.incidents, { incidents: advisorIncidents?.meta?.count })}
-                                    </span>
+                                    justifyContent={{ default: 'justifyContentCenter' }}
+                                >
+                                    {advisorIncidentsStatus === 'fulfilled'
+                                        ? <React.Fragment>
+                                            {
+                                                advisorIncidents?.meta?.count > 0 &&
+                                                    <ExclamationCircleIcon className='pf-v5-u-font-size-xl pf-v5-u-danger-color-100 pf-u-mr-sm' />
+                                            }
+                                            <span className='pf-v5-u-font-size-2xl pf-v5-u-text-align-center pf-v5-u-font-weight-normal'>
+                                                {intl.formatMessage(messages.incidents, { incidents: advisorIncidents?.meta?.count })}
```

Comment:
Here we can't use optional chaining given that the value is present
```suggestion
                                                {intl.formatMessage(messages.incidents, { incidents: advisorIncidents.meta.count })}
```